docker/docker-compose.1604.53.yaml

```diff
@@ -35,6 +35,9 @@ services:
       - MAX_ALLOCS_ALLOWED_read_10000_chunks_from_file=200500 #5.3 improvement 210050
       - MAX_ALLOCS_ALLOWED_udp_1000_reqs_1_conn=16250
       - MAX_ALLOCS_ALLOWED_udp_1_reqs_1000_conn=226050
+      - MAX_ALLOCS_ALLOWED_1000_reqs=6100
```

Comment:
isn't `1000_reqs` similar to `ping_pong_1000_reqs_1_conn`?

Reply:
Yeah - I'll ditch this one.

Reply:
which in fact makes the other problems go away.